aws-observability/application-signals-demo
# main

- code working
- update service and deployment names with language suffix
- fix python issue, tested ok
- create sns topic
- Updated OneStep Script and added DB deployment to EKS
- fix restTemplate call from customer service
- made some fixes
- change from sns to dynamodb
- updated synthetic canaries
- updated readme for ec2
- cleanup
- updated canaries
- added liveness probe for python services
- terraform update
- fix terraform and setup scripts issue, slo still not working
- applied some comments to clean up
- using python ecr images for docker
- updated readme
- update for checkov on terraform
- Fixed SLO creation script (#19)
- address comments
- remove unnecessary code
- avoid user creation
- fix readme description
- fix readme description
- fix readme code block issues
- main (#20)
- Changed replica count to 1 for customers service and updated canary
- Updated cleanup script to avoid deadlock when deleting persistent volume.

*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
